### PR TITLE
[BugFix] Update MyMessage options menu to correct size/visibility on mobile browsers

### PR DESCRIPTION
**Pull request by emlynmac**
# What
Ensure that menu options are the correct size and consistent

# Why
Refactor to fluent-contrib-chat meant some changes

# How Tested
Storybook; browser emulation of mobile

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/react-components/src/components/ChatMessage/ChatMessageComponentAsMessageBubble.tsx [hunk 1]
```diff
@@ -338,20 +334,6 @@ const MessageBubble = (props: ChatMessageComponentAsMessageBubbleProps): JSX.Ele
...
               className: chatMyMessageStyles.root,
-              onBlur: (e) => {
-                // copy behavior from North*
-                // `focused` controls is focused the whole `ChatMessage` or any of its children. When we're navigating
-                // with keyboard the focused element will be changed and there is no way to use `:focus` selector
-                const shouldPreserveFocusState = e.currentTarget.contains(e.relatedTarget);
-
-                setFocused(shouldPreserveFocusState);
-              },
-              onFocus: () => {
```
Comment (vhuseinova-msft) [inline packages/react-components/src/components/ChatMessage/ChatMessageComponentAsMessageBubble.tsx:349]: it looks like I can't navigate to the menu button anymore with Tab and Accessibility turned on

File: packages/react-components/src/components/ChatMessage/ChatMessageComponentAsMessageBubble.tsx [hunk 2]
```diff
@@ -192,22 +190,20 @@ const MessageBubble = (props: ChatMessageComponentAsMessageBubbleProps): JSX.Ele
...
-        forceShow: chatMessageActionFlyoutTarget === messageActionButtonRef,
-        onActionButtonClick: () => {
-          if (message.messageType === 'chat') {
-            props.onActionButtonClick(message, setMessageReadBy);
-            setChatMessageActionFlyoutTarget(messageActionButtonRef);
-          }
-        },
-        theme
-      });
+  const actionMenuProps = chatMessageActionMenuProps({
```
Comment (vhuseinova-msft) [inline packages/react-components/src/components/ChatMessage/ChatMessageComponentAsMessageBubble.tsx:193]: iOS: The second tap on message (not menu button) opens the edit menu. Is it expected?

File: packages/react-components/src/components/ChatMessage/ChatMessageComponentAsMessageBubble.tsx (hunk 1 above)
Comment (emlynmac) [reply packages/react-components/src/components/ChatMessage/ChatMessageComponentAsMessageBubble.tsx]: This looks this is something that was introduced between 27 Sept and now on main; also reproducible there.

**Review by palatter — APPROVED** [review]
Looks good. Tested on Safari iOS 17.

**Review by vhuseinova-msft — APPROVED** [review]

File: packages/react-components/src/components/ChatMessage/ChatMessageComponentAsMessageBubble.tsx (hunk 2 above)
Comment (vhuseinova-msft) [reply packages/react-components/src/components/ChatMessage/ChatMessageComponentAsMessageBubble.tsx:195]: To be addressed in the separate PR

**Comment by vhuseinova-msft** [comment]
> Edit menu is reappeared when message is tapped, I'm wondering if it expected. Steps:
> 
> * click on menu button (the edit menu will be shown)
> * click on message -> the edit menu is hidden and shown immediately
> 
> Also, if the menu button is clicked again, the position of the edit menu is changed for a few pixels horizontally

To be addressed in a separate PR

File: packages/react-components/src/components/ChatMessage/ChatMessageComponentAsMessageBubble.tsx (hunk 2 above)
Comment (emlynmac) [reply packages/react-components/src/components/ChatMessage/ChatMessageComponentAsMessageBubble.tsx:195]: Created separate issue for tracking this.

**Pull request merged**